database/sql/V008__contracts_upgrade.sql

```diff
@@ -0,0 +1 @@
+ALTER TYPE SigningScheme ADD VALUE 'ethsign';
```

Comment:
nit: the migration name should probably be "add ethsign signature or the like" (unless it goes in the same file as the contract upgrade)